css/styles.css

```diff
@@ -0,0 +1,33 @@
+.paragraph{
+    height: 300px;
+    width: 100%;
+    border-radius: 10px;
+    padding: 20px 20px 15px 15px;
+    color: #ffffff;
+}
+
+.p_1{
+    background-image: url("/imgs/blue.jpg");
+}
+
+.p_2{
+    background-image: url("/imgs/leaves.jpg");
+}
+
+.p_3{
+    background-image: url("/imgs/pink.jpg");
+    color: #000000;
+}
+
+@media only screen and (min-width: 769px) and (max-width: 1024px){
```

Comment:
you don't need to check for max-width: 1024px here. Since you are setting new styles in the next @media query, it will automatically pick those up as soon as browser screen size reaches 1024px wide